etc/yoda/component-config.json

```diff
@@ -1,5 +1,10 @@
 {
   "paths": ["/usr/yoda/lib/component"],
+  "stages": {
+    "base_component": ["turen.js", "lifetime.js", "light.js", "flora.js",
+      "dbus-registry.js", "wormhole.js"],
+    "next_tick": []
```

Comment:
How about the following:

```json
{
  "stages": {
    "prepare": [ "dbus-registry", "flora", "custodian" ],
    "connect": [ "tts", "turen", "light" ],
    "login": [ "rokid-voice-service" ],
    "ready": [ "lifetime", "wormhole" ]
  }
}
```

Reply:
And as for the runtime, the execution turn is also: prepare, connect, login and ready.